change/@internal-react-composites-b482f6b7-6a84-44ef-b79f-e7cd47c9570c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "hide dialpad when it's not PSTN",
```

Comment:
nit:
Change to "Dialpad hidden in non PSTN 1:1 Calls."

Reply:
updated